docs/docs/cmd/spo/tenant/tenant-site-membership-list.mdx

```diff
@@ -0,0 +1,93 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo tenant site membership list
+
+Retrieve information about default site groups' membership
+
+## Usage
+
+```sh
+m365 spo tenant site membership list [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the site.
+
+`-r, --role [role]`
+: Filter the results to include only users with the specified roles: 'Owner', 'Member', or 'Visitor'.
+
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To execute this command successfully, ensure you are logged in to the CLI for M365 with an account that has SharePoint admin permissions. For other scenarios, refer to the `spo web get --withGroups` and `spo group member list` commands.
+
+:::
+
+## Examples
+
+Retrieves information about default site groups' owners, members, and visitors of the site.
+
+```sh
+m365 spo tenant site membership list --siteUrl https://contoso.sharepoint.com
+```
+
+Retrieves information about site owners.
+
+```sh
+m365 spo tenant site membership list --siteUrl https://contoso.sharepoint.com --role Owner
+```
+
+## Response
```

Comment:
We're missing a CSV and MD response.

Reply:
Is it okay to add **No Response** in these cases? I guess that due to the use of an object instead of a list, these output types don't provide any valuable results.

Reply:
That's annoying. Could we alter the result a bit so we have some kind of output?

Reply:
Ok, I'm not sure if this is the best approach, but for the MD and CSV formats, I created a different output. Instead of returning an object, it provides a list of users with an `associatedGroupType` property to show which group a user is related to.